app.py

```diff
@@ -0,0 +1,40 @@
+
+from opentok import OpenTok
+from flask import Flask, request
+app = Flask(__name__)
+app.config.from_pyfile('.env')
+
+opentok = OpenTok(app.config.get('OPENTOK_API_KEY'), app.config.get('OPENTOK_API_SECRET'))
+
+
+@app.route('/')
+def index():
+    return "Success! Endpoints available: /session."
+
+
+@app.route('/session', methods=['POST'])
+def session():
+    # return "Success! Endpoints available: /session."
+    error = None
+    args = []
+    if request.method == 'POST':
```

Comment:
Are you going to add additional routes or an else condition if the route isn't a POST request?

Reply:
It already returns an HTTP 405 (method not allowed). Should be sufficient.